File: 프로그래머스/lv0/120839. 가위 바위 보/가위 바위 보.js
// 2 -> 0
// 0 -> 5
// 5 -> 2

function solution(rsp) {
    let ans = ''

   for (let i = 0; i < rsp.length; i++) {
        if (rsp[i] == 2) ans += 0
        if (rsp[i] == 0) ans += 5
        if (rsp[i] == 5) ans += 2
   }
   return ans
}

File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.java
class Solution {
  public String solution(String rsp) {
    StringBuilder sb = new StringBuilder();
    char[] arr = rsp.toCharArray();

    for (int i = 0; i < arr.length; i++) {
      char flag = arr[i];
      switch (flag) {
        case '2':
          sb.append('0');
          break;
        case '0':
          sb.append('5');
          break;
        case '5':
          sb.append('2');
          break;
      }
    }

    return sb.toString();
  }
}